tests/helpers/test_schema_config_entry_flow.py

```diff
@@ -27,6 +28,17 @@
 TEST_DOMAIN = "test"
 
 
+class TestSchemaConfigFlowHandler(SchemaConfigFlowHandler):
```

Comment:
How is this change related to the change in the data_entry_flow module?

Reply:
I think I added this change from #83546 by mistake. #83546 has been merged so these changes are gone with the rebase 🙈